src/NuGet.Services.AzureSearch/Db2AzureSearch/NewPackageRegistration.cs

```diff
@@ -9,25 +9,28 @@ namespace NuGet.Services.AzureSearch.Db2AzureSearch
 {
     /// <summary>
     /// The information required to bring an entire package registration up to date in the Azure Search indexes. This
-    /// data is populated from the database by db2azuresearch.
+    /// data is populated from the database and storage by db2azuresearch.
     /// </summary>
     public class NewPackageRegistration
     {
         public NewPackageRegistration(
             string packageId,
             long totalDownloadCount,
             string[] owners,
-            IReadOnlyList<Package> packages)
+            IReadOnlyList<Package> packages,
+            bool isExcludedByDefault = false)
```

Comment:
nit: no defaults here. the callers are minimal so it's best to be specific.

I followed this pattern everywhere in the Azure Search models because of the null / true / false case we spoke about yesterday.